### ISSUE TITLE
Interfaces with multiple wildcard arguments give incorrect lint

### ISSUE THREAD

**Issue by isaacabraham** [fsprojects/FSharpLint#154]
Given an interface that has a method that requires two arguments, if you call it as folliows you are shown an incorrect lint: -

```fsharp
{ new IFoo with
   member __.Bar(_, _) = ()
```

The warning is ```... __.Bar(_,_) can be replaced with __.Bar(_)```. But with interfaces you cannot do this - you have to provide both arguments as separate wildcards.

**Comment by duckmatt** [comment]
Thanks for the report. Are you using the latest version? It has been fixed before here: https://github.com/fsprojects/FSharpLint/issues/145

**Comment by isaacabraham** [comment]
OK - I'm using VFPT to get access to FSharpLint - I can only assume that this has not yet made it downstream cc: @dungpa

**Comment by duckmatt** [comment]
Ah, that would explain it, VFPT is currently on 0.0.13 and the fix went into 0.0.14
